float64 latitude
float64 longitude
int64 fix
int64 number_of_satelites
float64 horizontal_precision
float64 altitude
uint64 timestamp